src/pages/Homepage/BookCatalogueSection/BookCatalogueSection.module.css

```
@@ -149,3 +149,14 @@
     width: 100%;
   }
 }
+
+.books {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 10rem;
```

```suggestion
  row-gap: 120px;
  column-gap: 40px;
```
This gap should be the same as in Figma for catalogue section.